job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -35,6 +35,11 @@ def SERVERLESS_LOGIC_DROOLS_CURRENT_PRODUCT_VERSION='8.26.0'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_BRANCH='main'
 def SERVERLESS_LOGIC_CURRENT_PRODUCT_CONFIG_BRANCH="master"
 
+def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='1.25.0'
+def SERVERLESS_LOGIC_DROOLS_NEXT_PRODUCT_VERSION='8.24.0'
+def SERVERLESS_LOGIC_NEXT_PRODUCT_BRANCH='1.27.x'
+def SERVERLESS_LOGIC_NEXT_PRODUCT_CONFIG_BRANCH="openshift-serverless-logic/1.27.x"
```

Comment:
```suggestion
def SERVERLESS_LOGIC_NEXT_PRODUCT_CONFIG_BRANCH="openshift-serverless-logic/1.25.x"
```

Since product is 1.25.x...

Reply:
For now the created branch is openshift-serverless-logic/1.27.x